styles/mainStyle.css

```diff
@@ -858,34 +857,521 @@ header {
   }
 }
 
-@media screen and (min-width: 672px) and (max-width: 1000px) {
+.contact_section {
+  margin-top: 120px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.contact_header {
+  font-weight: 700;
+  font-family: Inter, sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  font-style: normal;
+  color: #172b4d;
+  width: 327px;
+}
+
+.contact_form {
+  margin-top: 23px;
+  margin-right: 24px;
+  margin-left: 24px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.input1,
+.input2 {
+  border-bottom: 1px solid #dbd8d7;
+  width: 327px;
+  font-family: inter, sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  padding: 14px;
+}
+
+.input1 {
+  margin-top: 73px;
+}
+
+.input2 {
+  margin-top: 39px;
+}
+
+.input3 {
+  background-color: #fbf8f7;
+  color: #3c3a39;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  width: 327px;
+  height: 114px;
+  margin-top: 24px;
+  padding: 14px;
+}
+
+.contact_btn {
+  background-color: #ff6b00;
+  color: #fff;
+  font-family: inter, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 3%;
+  text-align: center;
+  padding: 12px;
+  margin-top: 24px;
+  margin-bottom: 124px;
+  width: 131px;
+}
+
+.contact_btn:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+.contact_btn:active {
+  background-color: #e05304;
+  color: #f4f1f0;
+}
+
+.header_title {
+  position: absolute;
+  background: #3c3a39;
+  height: 92px;
+  display: flex;
+  align-items: center;
+  justify-self: center;
+  justify-content: space-between;
+}
+
+.port_name {
+  width: 50%;
+  height: 24px;
+  margin-left: 12px;
+  margin-top: 10px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: flex-start;
+  color: #fff;
+}
+
+.port_nav {
+  display: flex;
+  padding-left: 20px;
+  margin-left: 16.67%;
+  justify-content: flex-end;
+  align-items: flex-end;
+}
+
+#navigation {
+  display: none;
+}
+
+@media screen and (min-width: 992px) and (max-width: 1100px) {
+  /* Frame and toolbar styling */
```

Comment:
- [ ] [OPTIONAL]  I have noticed that you added (max-width: 1100px) properties. However this is making issues, because for example my desktop width is wider than 1100, and therefore, sadly, I am getting a mobile navbar:
![image](https://user-images.githubusercontent.com/77038610/149546444-739e850b-deab-49ac-a143-1cf2b0939c59.png)
> I would recommend you to delete `max-width:1100 `property